tapalcatl_server/handler.go

```diff
@@ -468,7 +471,14 @@ func MetatileHandler(p Parser, metatileSize int, mimeMap map[string]string, stor
 		reqState.Coord = &parseResult.Coord
 		reqState.HttpData = &parseResult.HttpData
 
-		metaCoord, offset := parseResult.Coord.MetaAndOffset(metatileSize)
+		metaCoord, offset, err := parseResult.Coord.MetaAndOffset(metatileSize, tileSize)
+		if err != nil {
+			configErrors.Add(1)
+			logger.Warning(LogCategory_ConfigError, "MetaAndOffset could not be calculated: %s", err.Error())
+			http.Error(rw, err.Error(), http.StatusInternalServerError)
+			reqState.ResponseState = ResponseState_Error
+			return
```

Comment:
There are other legitimate reasons why we wouldn't have a fetch state, and think making `FetchState_Nil` exempt is the right thing to do. I thought that's how it already worked :)